Sources/UInt128.swift

```diff
@@ -339,30 +300,21 @@ extension UInt128: UnsignedInteger {
         self.init(value.toUIntMax())
     }
     public func toUIntMax() -> UIntMax {
-        return UIntMax(value.lowerBits)
+        return lo.toUIntMax()
     }
     // MARK: Hashable Conformance
     public var hashValue: Int {
-        return Int(
-            value.lowerBits.hashValue ^ value.upperBits.hashValue
-        )
-    }
-    // MARK: ForwardIndexType Conformance
-    public func successor() -> UInt128 {
-        return self &+ 1
-    }
-    // MARK: BidirectionalIndexType Conformance
-    public func predecessor() -> UInt128 {
-        return self &- 1
+        return lo.hashValue ^ hi.hashValue
     }
 }
+
 // MARK: - Strideable
 extension UInt128: Strideable {
     public typealias Stride = Int
     /// Returns an instance of UInt128 that is the current instance's
     /// value increased by `n` when `n` is positive or decreased
     /// by `n` when `n` is negative.
-    public func advancedBy(_ n: Stride) -> UInt128 {
```

Comment:
👎 

This needs to stay until a major version bump is made, as it is removing parts of the API that users might be relying on.